lib/pages/faq/faqPage.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:acs_upb_mobile/pages/faq/model/question.dart';
+import 'package:acs_upb_mobile/pages/faq/service/question_service.dart';
+import 'QuestionsList.dart';
```

Comment:
Imports should be absolute (`import 'package:acs_upb_mobile...`). It's usually the easiest to just let AS do the imports for you.